vm/src/vm.rs

```diff
@@ -185,6 +207,8 @@ impl VirtualMachine {
             settings,
             recursion_limit: Cell::new(512),
             codec_registry: RefCell::default(),
+            repl: RefCell::new(repl),
+            repl_history_path,
```

Comment:
Detail about force commit:
`repl_history_path: repl_history_path` to `repl_history_path`